File: C#1/03-Operators-and-Expressions-Homework/04Rectangles/Rectangles.cs
using System;
using System.Threading;
using System.Globalization;

//Write an expression that calculates rectangle’s perimeter and area by given 
//width and height. 
//The width and height should be read from the console.

//input : 
//The input will consist of 2 lines: On the first line, you will receive a 
//floating-point number that will represent the width of the rectangle.
//On the second line, you will receive another floating-point number that will 
//represent the height of the rectangle.

//output :
//a single line - the rectangle's perimeter and area, separated by a whitespace.
//Print the perimeter and area values with exactly 2-digits precision after 
//the floating point



class Rectangles
{
        static void Main()
        {
            Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;

            float width = float.Parse(Console.ReadLine());
            float height = float.Parse(Console.ReadLine());

            float permiter = 2 * (width + height);
            float area = width * height;

            Console.WriteLine("{0:0.00} {1:0.00}", area, permiter);
        }
}

File: JS/HW/Operators-and-Expressions/rectangles.js
//Implement a javascript function that takes an argument array as a parameter and 
//calculates rectangle’s area and perimeter by given width and height. 
//The width and height will both be passed in the argument array.

function solve(args){
    let width = +args[0];
    let height = +args[1];

    let perimeter = 2*(width + height);
    let area = width * height;

    console.log(area.toFixed(2) + " " + perimeter.toFixed(2));
}

solve(['2.5', '3'] );
solve(['5', '5'] );
solve(['3', '4'] );